src/EnergyPlus/Psychrometrics.hh

```diff
@@ -334,8 +334,8 @@ namespace Psychrometrics {
         return 1.00484e3 * TDB + dW * (2.50094e6 + 1.85895e3 * TDB); // enthalpy {J/kg}
     }
 
-    inline Real64 PsyCpAirFnWTdb(Real64 const dw, // humidity ratio {kgWater/kgDryAir}
-                                 Real64 const T   // input temperature {Celsius}
+    inline Real64 PsyCpAirFnWTdb(Real64 const dw,          // humidity ratio {kgWater/kgDryAir}
+                                 Real64 const EP_UNUSED(T) // input temperature {Celsius}
```

Comment:
My preference would be to go ahead and just remove this argument entirely if it isn't used anymore.

Reply:
Removed the unused argument (TEMPERATURE) from PsyCpAirFnWTdb() function. Run unit tests locally. All happy.